Bridges/HttpKernel.php

```diff
@@ -78,17 +78,20 @@ public function onRequest(ReactRequest $request, HttpResponse $response)
 
         $syRequest = $this->mapRequest($request);
 
-        //start buffering the output, so cgi is not sending any http headers
-        //this is necessary because it would break session handling since
-        //headers_sent() returns true if any unbuffered output reaches cgi stdout.
-        ob_start();
-
         try {
+            // start buffering the output, so cgi is not sending any http headers
+            // this is necessary because it would break session handling since
+            // headers_sent() returns true if any unbuffered output reaches cgi stdout.
+            ob_start();
+
             if ($this->bootstrap instanceof HooksInterface) {
                 $this->bootstrap->preHandle($this->application);
             }
 
             $syResponse = $this->application->handle($syRequest);
+
+			// should not receive output from application->handle()
+            ob_end_clean();
         } catch (\Exception $exception) {
```

Comment:
> 

We need a ob_end_clean call also in catch block I think. I guess its better to just let ob_start out of try and add
ob_end_clean after the whole try-catch.

> Btw, tabs detected :P

@marcj Will fix this. Could you also comment on the content-length?